src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -456,4 +470,87 @@ tnsr::a<DataType, SpatialDim, Frame> spacetime_deriv_of_norm_of_shift(
     const tnsr::iaa<DataType, SpatialDim, Frame>& phi,
     const tnsr::aa<DataType, SpatialDim, Frame>& pi) noexcept;
 // @}
+
+namespace Tags {
+/*!
+ * \ingroup GeneralRelativityGroup
```

Comment:
We don't put the tags in the groups so as to not clutter the groups with tags. It's easier to have the Tags in a namespace (which they're already always in :) )